src/Strong_collapse/doc/Intro_strong_collapse.h

```diff
@@ -0,0 +1,115 @@
+ * There is a <b>strong collapse</b> from a simplicial complex \f$K\f$ to its subcomplex \f$L\f$, 
+ * if there exists a series of elementary strong collapses from \f$K\f$ to \f$L\f$, denoted as \f$K\f$
+ * \f${\searrow\searrow}\f$ \f$L\f$.
+ * 
+ * A strong collapse is a homotopy preserving operation,in fact it a special type of the classical simple collapse. 
+ * A complex without any dominated vertex is called a minimal complex and the core \f$K^0\f$ of simplicial comlex is a
+ * minimal complex such that \f$K\f$ \f${\searrow\searrow}\f$ \f$K^0\f$.
+ * Computation of a core involves computation of dominated vertices and the dominated vertices can be easily
+ * characterized as follows:
+ * 
+ * -- For general simplicial complex: A vertex \f$v \in K\f$ is dominated by another vertex \f$v^{\prime} \in K\f$,
+ * <i>if and only if</i> all the maximal simplices of \f$K\f$ that contain $v$ also contain \f$v^{\prime}\f$
+ * 
+ * -- For a flag complex: A vertex \f$v \in K\f$ is dominated by another vertex \f$v^{\prime} \in K\f$, <i>if and only
+ * if</i> all the vertices in \f$K\f$ that has an edge with $v$ also has an edge with \f$v^{\prime}\f$, given 
+ * \f$[v.v^\prime]\f$ is an egde \f$\in K\f$.
+ *  
+ * This module implements strong collapse of a flag complex, in particular  Vietoris-Rips (VR) complex from its graph
+ * (1-skeleton) based on the above domination criteria. The algorithm to compute the core is described in Section 3
+ * \cite strongcollapsesocg2019. 
+ * \subsection persitence_computation_via_strong_collapse Persitence computation via strong_collapse
+ * Strong collapse can be successfully employed to reduce a given sequence of simplicial complexes to a smaller induced
+ * sequence which preserves the persistent homology of the original sequence. 
+ * The general idea is to compute the cores of each complex in the sequence and there exists an induced simplicial maps
+ * among the cores, which preserves the persistent homology. 
+ * The precise mechanism for this reduction has been described in Section 4 \cite strongcollapseesa2018. 
+ * Here we implement this mechanism for a filtration of Rips complex, where a filtration is a sequence of simplicial
+ * (here Rips) complexes connected with inclusions. 
+ * After perfoming the reduction the filtration reduces to a flag-tower, which is a sequence of flag complexes
+ * connected through more general (inclusion and contractions) simplicial maps. 
+ * However to compute the persistent homology of any tower one needs to convert it to an equivalent filtration. Section
+ * 4 of \cite strongcollapsesocg2019 descibes an efficient strategy to convert a flag tower to a flag filtration, 
+ * using only the 1-skeletons of the complexes, which we implement here.
+ * 
+ * \subsection strong_collapse_from_points_example Example from a point cloud and a distance function
+ * 
+ * This example builds the edge graph from the given points, threshold value, and distance function.
```

Comment:
Without context, I have no idea what the threshold value is about. Looking at it, it appears that it is a maximum on the edge lengths that we consider?